.github/workflows/Testing.yaml

```
@@ -11,25 +11,48 @@ on:
 jobs:
   test:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.nightly }}
     strategy:
+      fail-fast: false
       matrix:
         version:
           - '1.0'
           - '1.1'
           - '1.2'
           - '1.3'
           - '1.4'
-          - 'nightly'
         os:
           - ubuntu-latest
           - windows-latest
           - macOS-latest
         arch:
           - x64
           - x86
+        nightly: [false]
```

the keyword `nightly` is ambiguous, maybe replace with:

```suggestion
        is_nightly: [false]
```

---

done.